hybridauth.module

```diff
@@ -12,8 +12,235 @@ function hybridauth_form_alter(&$form, FormStateInterface $form_state, $form_id)
     $form['hybridauth'] = [
       '#title' => t('Log in with Linkedin'),
       '#type' => 'link',
-      '#url' => Url::fromRoute('hybridauth.authenticate', ['provider_id' => 'linkedin']),
+      '#url' => Url::fromRoute('hybridauth.authenticate', ['provider_id' => 'LinkedIn']),
     ];
+
+    $form['hybridauth_facebook'] = [
```

Comment:
this should be autogenerated based on enabled providers. Currently, it's hardcoded.

Reply:
Done. Thank you for idea!